src/NuGet.Clients/NuGet.SolutionRestoreManager/InfoBarService.cs

```diff
@@ -0,0 +1,158 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
```

Comment:
Can we `#nullable enable` this new service? I realize most of the VS APIs don't have null checking enabled, but at least it may help our own service's readability in a couple spots.